samples/tab-adaptive-cards/nodejs/Readme.md

```diff
@@ -0,0 +1,123 @@
+
+2. To test locally, [NodeJS](https://nodejs.org/en/download/) must be installed on your development machine (version 10.14 or higher).
+
+    ```bash
+    # determine node version
+    node --version
+    ```
+
+3. To test locally, you'll need [Ngrok](https://ngrok.com/) installed on your development machine.
+Make sure you've downloaded and installed Ngrok on your local machine. ngrok will tunnel requests from the Internet to your local computer and terminate the SSL connection from Teams.
+
+> NOTE: The free ngrok plan will generate a new URL every time you run it, which requires you to update your Azure AD registration, the Teams app manifest, and the project configuration. A paid account with a permanent ngrok URL is recommended.
+
+## To try this sample
+
+- Register Azure AD applications
+    -   Register your bot using bot channel registration in Azure AD portal, following the instructions [here](Wiki/azure-bot-channels-registration.md).
+
+- Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+
+- In a console, navigate to `samples/tab-adaptive-cards/nodejs`
+
+    ```bash
+    cd samples/tab-adaptive-cards/nodejs
+    ```
+
+- Run ngrok - point to port `3978`
+
+    ```bash
+    ngrok http -host-header=localhost 3978
+    ```
+
+
+- Update the `.env` configuration for the bot to use the `MicrosoftAppId` (Microsoft App Id) and `MicrosoftAppPassword` (App Password) from the Bot Framework registration. 
+Also update `BaseUrl` according to your code runtime environment and `ConnectionName` with Bot Channel Registration connection name.
+> NOTE: the App Password is referred to as the `client secret` in the azure portal and you can always create a new client secret anytime.
+
+- Install modules & Run the `NodeJS` Server 
+    - Server will run on PORT:  `3978`
+    - Open a terminal and navigate to project root directory
+    
+    ```bash
+    npm run server
+    ```
+    
+    > **This command is equivalent to:**
+    _npm install  > npm start_
+
+- __*This step is specific to Teams.*__
+    - **Edit** the `manifest.json` contained in the  `appPackage` folder to replace your Microsoft App Id (that was created when you registered your bot earlier) *everywhere* you see the place holder string `<<YOUR-MICROSOFT-APP-ID>>` (depending on the scenario the Microsoft App Id may occur multiple times in the `manifest.json`) also update the `<<DOMAIN-NAME>>` with the ngrok URL
+    - **Zip** up the contents of the `appPackage` folder to create a `manifest.zip`
+    - **Upload** the `manifest.zip` to Teams (in the Apps view click "Upload a custom app")
+
+## Interacting with the bot in Teams
+
+You can interact with this bot by `@Sequential Workflows` (BotName). The bot will respond with adaptive card requesting you the details.
+
+- Install App
+
+Navigate to `Manage apps` > `Upload a custom app` (Bottom-Right of the screen) > Upload `manifest.zip` > `Add`
```

Comment:
This looks like section for another sample. Could you please update the these steps for Tab?

Reply:
Changed